### PR TITLE
feat: Add warning message when tsconfig is invalid

### PR DESCRIPTION
**Pull request by citrus327**
Add warning message when tsconfig is invalid

### CODE REVIEW

File: src/options.ts [hunk 1]
```diff
@@ -33,6 +33,8 @@ export const getOptions = (
   // Only fallback to `jsconfig.json` when tsconfig can not be resolved AND custom tsconfig filename is not provided
   if (!result && !tsconfig) {
     result = getTsconfig(cwd, 'jsconfig.json');
+  } else {
+    console.warn('[rollup-plugin-swc] tsconfig does not exists, fallback to jsconfig.json');
   }
```
Comment (SukkaW) [inline src/options.ts]:
```suggestion
  if (!result && !tsconfig) {
    console.warn('[rollup-plugin-swc] tsconfig does not exist, looking for jsconfig.json instead');
    result = getTsconfig(cwd, 'jsconfig.json');
  }
```

**Review by SukkaW — CHANGES_REQUESTED** [review]

File: src/options.ts (hunk 1 above)
Comment (SukkaW) [reply src/options.ts]: Also, IMHO we should use rollup built-in log functions here: https://rollupjs.org/plugin-development/#this-warn

**Comment by citrus327** [comment]
@SukkaW I've updated the changes, please review again, thank you.

File: src/options.ts [hunk 2]
```diff
@@ -33,8 +35,7 @@ export const getOptions = (
   // Only fallback to `jsconfig.json` when tsconfig can not be resolved AND custom tsconfig filename is not provided
   if (!result && !tsconfig) {
     result = getTsconfig(cwd, 'jsconfig.json');
-  } else {
-    console.warn('[rollup-plugin-swc] tsconfig does not exists, fallback to jsconfig.json');
+    ctx.warn('tsconfig does not exists, fallback to jsconfig.json');
```
Comment (SukkaW) [inline src/options.ts]:
```suggestion
    ctx.warn({
      message: 'Can\'t find tsconfig.json, trying jsconfig.json now',
      pluginCode: 'SWC_TSCONFIG_NOT_EXISTS'
    });
```

**Pull request merged**